package.json

```diff
@@ -1,27 +1,32 @@
 {
-  "name": "wc3maptranslator",
+  "name": "wc3maptranslator-ts",
```

Comment:
Restore this to `wc3maptranslator`, I'm cautious about messing with the name since the project is already deployed to npm. We'll adjust the README and tagline to mention the TypeScript support.

Reply:
I am thinking here if we let the js files be compiled in parallel like the setting is right now. (In vs code I do not see the js files since I have "hidden them") Wouldn't it still work with npm just have some breaking changes therefore the 3.0.0 and all the typings are directly at the right place together with the transpiled ts -> js files.

Reply:
I mean that I'm not sure if npm will like the name being changed to add the `-ts` suffix. The name should remain as the original `wc3maptranslator`, since it's the same project.